examples/model_selection/plot_confusion_matrix.py

```diff
@@ -59,29 +57,15 @@ def plot_confusion_matrix(cm, classes,
     """
     if normalize:
         cm = cm.astype('float') / cm.sum(axis=1)[:, np.newaxis]
-        print("Normalized confusion matrix")
-    else:
-        print('Confusion matrix, without normalization')
 
+    print(title)
```

Comment:
I'm not entirely sure if this function is still helpful for copy & paste. It mostly labels the axis and does normalization, which might be helpful... I left it for now.